tools/diagnostics/nsdiff

```diff
@@ -0,0 +1,875 @@
+#!/usr/bin/env python
+
+# README:
+#
+# This script assumes:
```

Comment:
FROM ALEX:
Please specify what exactly is compared:

- txn log (for all ledgers)
- hash store (for all ledgers)
- all states
- state_signature (BLS)
- attr_store
- seq_no_db
- ts_store
- ????

Reply:
@ckochenower added some doc in the comments to address this.